Makefile

```diff
@@ -73,42 +73,46 @@ $(subst .doc/models/template,,$(MODELS_D)): .install/models/template
 $(subst .install/base/logger,,$(ALL_PKGS_I)): | .install/base/logger
 $(subst .doc/base/logger,,$(ALL_PKGS_D)): | .install/base/logger
 
-$(call depends,base/utils): | .install/base/remote
-$(call depends,base/db): | .install/base/utils
-$(call depends,base/qaqc): | .install/base/utils
-$(call depends,base/settings): | .install/base/utils .install/base/db
-$(call depends,base/visualization): | .install/base/db
-$(call depends,base/workflow): | .install/base/db .install/base/settings
-$(call depends,modules/allometry): | .install/base/db
-$(call depends,modules/assim.batch): | .install/base/utils .install/base/db .install/base/settings .install/base/remote .install/modules/meta.analysis
-$(call depends,modules/assim.sequential): | .install/base/remote
-$(call depends,modules/benchmark): | .install/base/db .install/base/remote .install/base/settings .install/base/utils .install/modules/data.land
-$(call depends,modules/data.atmosphere): | .install/base/utils .install/base/remote .install/base/db
-$(call depends,modules/data.hydrology): | .install/base/utils
-$(call depends,modules/data.land): | .install/base/db .install/base/utils .install/base/settings .install/base/remote
-$(call depends,modules/data.mining): | .install/base/utils
-$(call depends,modules/data.remote): | .install/base/remote
-$(call depends,modules/meta.analysis): | .install/base/utils .install/base/db .install/base/settings .install/modules/priors
-$(call depends,modules/priors): | .install/base/utils
-$(call depends,modules/rtm): | .install/modules/assim.batch .install/base/utils .install/models/ed
-$(call depends,modules/uncertainty): | .install/base/utils .install/base/db .install/modules/priors .install/modules/emulator
-$(call depends,models/biocro): | .install/mockery .install/base/utils .install/base/settings .install/base/db .install/modules/data.atmosphere .install/modules/data.land .install/base/remote
-$(call depends,models/cable): | .install/base/utils
-$(call depends,models/clm45): | .install/base/utils
-$(call depends,models/dalec): | .install/base/remote
+$(call depends,base/all): | .install/base/db .install/base/settings .install/modules/meta.analysis .install/base/logger .install/base/utils .install/modules/uncertainty .install/modules/data.atmosphere .install/modules/data.land .install/modules/data.remote .install/modules/assim.batch .install/modules/emulator .install/modules/priors .install/modules/benchmark .install/base/remote .install/base/workflow
+$(call depends,base/db): | .install/base/logger .install/base/remote .install/base/utils
```

Comment:
All packages already depend on logger -- line 73 above is "call depends,base/logger on every package except logger itself".

Reply:
I now generate the list, so no more manually need to edit it. It is also in a separate file.